lib/samsam.test.js

```diff
@@ -3,13 +3,921 @@
 var referee = require("@sinonjs/referee");
 var assert = referee.assert;
 var refute = referee.refute;
+require("jsdom-global")();
+var samsam = require("./samsam");
 
-describe("SamSam", function () {
-    it("Should Always Be true", function () {
-        assert.equals(42, 42);
+describe("isElement", function () {
+    it("Should pass if DOM element node", function () {
+        if (typeof document !== "undefined") {
+            var element = document.createElement("div");
+            var checkElement = samsam.isElement(element);
+            assert.equals(checkElement, true);
+        }
     });
 
-    it("Should Always Be False", function () {
-        refute.equals(42, 12);
+    it("Should fail if DOM text node", function () {
+        if (typeof document !== "undefined") {
+            var element = document.createTextNode("Hello");
+            var checkElement = samsam.isElement(element);
+            refute.equals(checkElement, true);
+        }
+    });
+
+    it("Should fail if node like object", function () {
+        var nodeLike = { nodeType: 1 };
+        var checkElement = samsam.isElement(nodeLike);
+        refute.equals(checkElement, true);
+    });
+
+    it("Should fail if number", function () {
+        var checkElement = samsam.isElement(42);
+        refute.equals(checkElement, true);
+    });
+
+    it("Should fail if object", function () {
+        var checkElement = samsam.isElement({});
+        refute.equals(checkElement, true);
+    });
+});
+
+describe("isArguments", function () {
+    it("Should pass if arguments is an object", function () {
```

Comment:
Would you mind changing all the first arguments to `it` to start with lowercase?

(`Should` => `should`)

Otherwise, it'll look weird when Mocha combines the `describe` and `it` to form a sentence:

`isArguments Should pass if arguments is an object` vs. `isArguments should pass if arguments is an object`

Reply:
Done 👍